src/github.com/matrix-org/dendrite/roomserver/input/events.go

```diff
@@ -17,12 +29,16 @@ func processRoomEvent(db RoomEventDatabase, input api.InputRoomEvent) error {
 		return err
 	}
 
-	if err := db.StoreEvent(event); err != nil {
+	// Check that the event passes authentication checks.
```

Comment:
Technically it isn't just that, it also then converts those auth event IDs into auth event NIDs, right?